rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper_integration_test.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018, Bosch Software Innovations GmbH.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gmock/gmock.h>
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "rcutils/types.h"
+
+#include "../../../src/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.hpp"
```

Comment:
I know it's a test, but given that long include path, is there any use to having this wrapper file public?

Reply:
It will be made public in #31, because at that point we need it in other tests as well. But until then, I think it's cleaner to do it like this.